lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/driver/ContainerManager.java

```diff
@@ -144,55 +137,61 @@ public void onNext(final RemoteMessage<ReefServiceProtos.RuntimeErrorProto> valu
     LOG.log(Level.FINE, "Initialized Container Manager with {0} containers", capacity);
   }
 
-  private Collection<String> normalize(final Collection<String> rackNames) {
+  private static Collection<String> normalize(final Collection<String> rackNames) {
```

Comment:
The actual wrapped `normalize` method deserves  a short comment on what it means here to normalize a rack name.